package.json

```diff
@@ -46,7 +46,7 @@
     "react": "16.14.0",
     "react-dom": "16.14.0",
     "react-intl": "^5.25.1",
-    "react-paragon-topaz": "^1.7.8",
+    "react-paragon-topaz": "1.8.0",
```

Comment:
@01001110J Why is this upgrade necessary?

Reply:
In this version is implemented the sticky sidebar, so I decided to make the upgrade now instead of wait for have more content and make the upgrade in the future.